appengine/standard/firebase/firetactoe/firetactoe.py

```diff
@@ -75,17 +77,24 @@ def _get_http(_memo={}):
         creds.authorize(http)
         _memo['http'] = http
     return _memo['http']
+# [END authed_http]
 
-
+# [START send_msg]
```

Comment:
Let's let Jim decide. As you noted in email, we probably can't get rid of them all.